features/networking/sriov.feature

```diff
@@ -14,3 +14,58 @@ Feature: Sriov related scenarios
       | app=sriov-network-config-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=sriov-network-operator     |
+
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-24702
+  @destructive
+  @admin
+  Scenario: netdevice VF for XXV710 can be worked well when sriovnetworknodepolicies is created for rhcos node
+    Given the sriov operator is running well
+    #When I run the :create admin command with:
+    #  | f | <%= BushSlicer::HOME %>/testdata/networking/sriov/sriovnetworkpolicy/intel-netdevice.yaml |
+    #Then the step should succeed
+    Given I create resource with following:
+       | cr_yaml       |  <%= BushSlicer::HOME %>/testdata/networking/sriov/sriovnetworkpolicy/intel-netdevice.yaml |
+       | cr_name       | intel-netdevice                                                                            |
+       | resource_type | sriovnetworknodepolicies                                                                   |
+    Then the step should succeed
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource    | sriovnetworknodestates           |
+      | namespace   | openshift-sriov-network-operator |
+      | o           | yaml                             |
+    Then the step should succeed
+    And the output should contain "vfID: 4"
+    """
+    Given I switch to the first user
+    And I have a project
+    And evaluation of `project.name` is stored in the :usr_project clipboard
+    Given I create resource with following:
+       | cr_yaml       |  <%= BushSlicer::HOME %>/testdata/networking/sriov/sriovnetwork/static-sriovnetwork.yaml |
+       | cr_name       | static-sriovnetwork                                                                      |
+       | resource_type | sriovnetwork                                                                             |
+       | project       | <%= cb.usr_project%>                                                                     |
+    Then the step should succeed
+
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource  | net-attach-def        |
+      | namespace | <%= cb.usr_project%>  |
+    Then the step should succeed
+    And the output should contain "static-sriovnetwork"
+    """ 
+    Given I switch to the first user
+    And I use the "<%= cb.usr_project%>" project
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/networking/sriov/pod/sriov-without-resource-static.yaml |
```

Comment:
@zhaozhanqi pls use Given I obtain... to get the testdata going forward.

Reply:
updated. thanks